add-chain/cmd/update-configs.go

```diff
@@ -25,6 +25,10 @@ var UpdateConfigsCmd = cli.Command{
 				panic(err)
 			}
 
+			chain.MaxSequencerDrift = 600
+			fifty := uint64(50)
+			chain.ChannelTimeoutBedrock = &fifty
```

Comment:
See below comment about use of this tool. 

Also general question about this tool: do we think it is useful enough to keep in the repo? Seems more like one-time maintenance code to me so might be a good candidate for removal.

Reply:
I've used this tool at least three times in the past few weeks to add new fields or update existing fields. Its much easier and more scalable to use the tool than manually adding/editing fields in every chain config in the registry. Maybe we can just improve the tool, and I can remove this diff to the tool in this pr, instead of removing it completely? Or is there a different way you have in mind to apply new settings to many chains at once?